src/Transport/HttpUnaryTransportTrait.php

```diff
@@ -92,10 +92,13 @@ private static function buildCommonHeaders(array $options)
 
         // If not already set, add an auth header to the request
         if (!isset($headers['Authorization']) && isset($options['credentialsWrapper'])) {
-            $bearerString = $options['credentialsWrapper']->getBearerString();
-            if (!empty($bearerString)) {
-                $headers['Authorization'] = $bearerString;
```

Comment:
The `authorization` header is lowercase in `updateMetadata` and uppercase here, so we should ensure this change is OK